script.js

```diff
@@ -249,6 +251,22 @@ window.GaiaAppIcon = (function(exports) {
     enumerable: true
   });
 
+  Object.defineProperty(proto, 'showName', {
+    get: function() {
+      return this._showName;
+    },
+
+    set: function(show) {
+      show = show ? true : false;
+      if (this._showName !== show) {
+        this._showName = show;
+        this._titleContainer.classList.toggle('hidden', !show);
```

Comment:
We discussed this on IRC. Short answer is 'yes', long answer is 'I forgot about the hidden property, but yes' :)